services/brig/src/Brig/API/OAuth.hs

```diff
@@ -175,14 +173,14 @@ verifyRefreshToken key rt = do
 createAccessTokenWithAuthorizationCode :: (Member Now r, Member Jwk r) => OAuthAccessTokenRequest -> (Handler r) OAuthAccessTokenResponse
 createAccessTokenWithAuthorizationCode req = do
   unless (req.grantType == OAuthGrantTypeAuthorizationCode) $ throwStd $ errorToWai @'OAuthInvalidGrantType
-  (cid, uid, scope, uri) <-
-    lift (wrapClient $ lookupAndDeleteByOAuthAuthorizationCode req.code)
+  (cid, uid, scope, uri, mChal) <-
+    lift (wrapClient $ lookupAndDeleteByOAuthAuthorizationCode (req.code))
```

Comment:
Interesting, why do we need the parentheses around `req.code`?

Reply:
We don't. Was probably due to a merge issue.